common/disabled/orderedCollection.go

```diff
@@ -0,0 +1,57 @@
+package disabled
+
+import (
+	"errors"
+)
+
+type orderedCollection struct {
+}
+
+var errDisabledComponent = errors.New("")
+
+// NewOrderedCollection creates a new ordered collection
+func NewOrderedCollection() *orderedCollection {
+	return &orderedCollection{}
+}
+
+// Add adds a new item to the order collector
```

Comment:
to the ordered collection

but the comments can be renamed to `// Function does nothing` or `// Function returns an error since this is a disabled component`

Reply:
changed all the comments